File: ontrack-extension-git/src/main/java/net/nemerosa/ontrack/extension/git/property/AbstractGitProjectConfigurationPropertyType.java
package net.nemerosa.ontrack.extension.git.property;

import net.nemerosa.ontrack.extension.support.AbstractPropertyType;
import net.nemerosa.ontrack.model.extension.ExtensionFeature;

public abstract class AbstractGitProjectConfigurationPropertyType<T> extends AbstractPropertyType<T> {

    protected AbstractGitProjectConfigurationPropertyType(ExtensionFeature extensionFeature) {
        super(extensionFeature);
    }

}

File: ontrack-extension-git/src/main/java/net/nemerosa/ontrack/extension/git/property/AbstractGitProjectConfigurationPropertyType.kt
package net.nemerosa.ontrack.extension.git.property

import net.nemerosa.ontrack.extension.support.AbstractPropertyType
import net.nemerosa.ontrack.model.extension.ExtensionFeature

abstract class AbstractGitProjectConfigurationPropertyType<T>
protected constructor(extensionFeature: ExtensionFeature) :
    AbstractPropertyType<T>(extensionFeature)
